nutri-track-bot/src/main/java/com/nutritrack/bot/service/telegram/factory/TelegramFactory.java

```diff
@@ -29,11 +29,15 @@ public static TelegramWebhookBot webhookBot(String token,
         return new TelegramWebhookBot(token) {
 
             {
-                registerCommands();
+                registerCommands("en");
             }
 
             @Override
             public BotApiMethod onWebhookUpdateReceived(Update update) {
+                if (update.hasMessage() && update.getMessage().getFrom() != null) {
```

Comment:
A request may contain different types of messages, and as a result, the language code might not always be present in the Message object. See example in com.nutritrack.bot.service.parser.impl.UpdateParserImpl#createCustomUpdate